### ISSUE TITLE
Product images not visible after sample data deploy

### ISSUE THREAD

**Issue by bhushanmeetanshi** [markshust/docker-magento#1194]
**Description**
Product Images not visible after sample data deploy

**Steps To Reproduce**
1. execute below automate script
```
# Create your project directory then go into it:
mkdir -p ~/Sites/magento
cd $_

# Run this automated one-liner from the directory you want to install your project.
curl -s https://raw.githubusercontent.com/markshust/docker-magento/master/lib/onelinesetup | bash -s -- magento.test 2.4.7 community

```
2. Install sample data 
```
bin/magento sampledata:deploy
bin/magento setup:upgrade
```

3. check images

**Expected Result**
Product images should be visible to the user

**Actual Result**
Product images not visible after sample data deploy

**Comment by bhushanmeetanshi** [comment]
![image](https://github.com/user-attachments/assets/e92f704e-ead2-4a6e-b653-b41412173046)

**Comment by bhushanmeetanshi** [comment]
unable to execute: `bin/magento catalog:images:resize` command

![image](https://github.com/user-attachments/assets/2c8fbb55-2806-4d0c-8acc-7826d18778f2)

**Comment by bhushanmeetanshi** [comment]
@YevhenZvieriev unable to view product images of sample data sometimes.

how to solve this problem. Can you please help me regarding with this problem?

**Comment by bhushanmeetanshi** [comment]
unable to execute: bin/magento catalog:images:resize command

**Comment by bhushanmeetanshi** [comment]
i have set two themes luma and hyva theme with two different website.

**Comment by bhushanmeetanshi** [comment]
@markshust or @YevhenZvieriev could u please guide me for this problem?

**Comment by YevhenZvieriev** [comment]
Hello, @bhushanmeetanshi 

Every time I encountered an image disappearing from the products, I reinstalled the Magento sandbox instance.

It took less than 10 minutes for me, and it is faster than looking to resolve this issue.

Honestly, I don't know why it is happening, but it may be because of ElasticSearch/OpenSearch(Should check logs), or sample data image access.

Magento devs recommend just executing `bin/magento catalog:images:resize`, but it doesn`t work for me.

On the Internet very little information about this...
